src/Microsoft.Performance.Toolkit.Engine/Engine.cs

```diff
@@ -287,6 +287,9 @@ public static Engine Create(IDataSource dataSource)
         /// <exception cref="ObjectDisposedException">
         ///     This instance is disposed.
         /// </exception>
+        /// <exception cref="NotSupportedException">
+        ///     The <c>createInfo.OptionsResolver.</c> are not supported by the provided Processing Source.
+        /// </exception>
         public static Engine Create(
```

Comment:
If this is generating a default EngineCreateInfo, this should never be thrown, should it?

Same for the next method.

Reply:
Good catch :). Thanks.